src/applications/vaos/services/appointment/transformers.js

```diff
@@ -152,10 +152,10 @@ function getMomentConfirmedDate(appt) {
   }
 
   const timezone = getTimezoneBySystemId(appt.facilityId)?.timezone;
-  const date = isVideoVisit(appt)
-    ? appt.vvsAppointments[0].dateTime
```

Comment:
There's no reason for us to use `dateTime` here, I've been meaning to switch it to `startDate`